theme/mixins/backdrop.less

```diff
@@ -0,0 +1,11 @@
+.backdrop() {
+  position: fixed;
+  z-index: 0;
```

Comment:
Is this `z-index` required? If so should we add an entry into [our z-index hierarchy](https://github.com/seek-oss/seek-style-guide/blob/master/theme/layout/z-indexes.less)?